doc/toolbox.1.md

```diff
@@ -87,40 +87,18 @@ by option **log-level**.
 Same as `--log-level=debug`. Use `-vv` to include `--log-podman`.
 
 ## COMMANDS
-
-Commands for working with toolbox containers and images:
-
-**toolbox-create(1)**
-
-Create a new toolbox container.
-
-**toolbox-enter(1)**
-
-Enter a toolbox container for interactive use.
-
-**toolbox-help(1)**
-
-Display help information about Toolbox.
-
-**toolbox-init-container(1)**
-
-Initialize a running container.
-
-**toolbox-list(1)**
-
-List existing toolbox containers and images.
-
-**toolbox-rm(1)**
-
-Remove one or more toolbox containers.
-
-**toolbox-rmi(1)**
-
-Remove one or more toolbox images.
-
-**toolbox-run(1)**
-
-Run a command in an existing toolbox container.
+The following commands correspond to markdown files which will help you with execution.
```

Comment:
This looks a bit odd.  :)

The user doesn't need to know if the manuals are written in Markdown or something else.  When the manuals are consumed through `man(1)` they will be in the troff format, and when they are consumed through the web browser they will be in HTML.

Notice how the [flatpak(1)](https://man7.org/linux/man-pages/man1/flatpak.1.html), [git(1)](https://git-scm.com/docs/git) or `podman(1)` manuals don't say anything like this.